src/dinitctl.cc

```diff
@@ -760,27 +760,106 @@ static void print_termination_details(int exit_status)
     }
 }
 
-// Wait for a service to reached stopped (do_stop == true) or started (do_stop == false) state.
-// Returns 0 if the service started/stopped, 1 if start/stop was cancelled or failed.
-static int wait_service_state(int socknum, cpbuffer_t &rbuffer, handle_t handle,
+// Process a SERVICEEVENT packet if it is related to the specified service handle, and optionally report
```

Comment:
See style guide, lines should wrap before 100 characters.

> Line widths - do not exceed 110 characters; try to limit to 100.

The only reason to go beyond 100 is if it is awkward to break the line beforehand or if it will naturally end just slightly over.